src/components/allComments.js

```diff
@@ -126,10 +149,11 @@ class AllComments extends React.Component {
         render={data => (
           <>
             {loading && <Typography>Loading...</Typography>}
-            {!loading && isSignedIn && (
+            {!loading && (
               <>
                 <Typography variant="h5">Comments</Typography>
                 {this.renderComments(allComments, data)}
+                {this.renderComments(apiComments, data)}
```

Comment:
With this `allComments` should be removed.

Reply:
okay then will be removed then